activerecord/test/cases/attribute_methods_test.rb

```diff
@@ -16,7 +16,7 @@
 class AttributeMethodsTest < ActiveRecord::TestCase
   include InTimeZone
 
-  fixtures :topics, :developers, :companies, :computers
+  fixtures :companies, :computers, :developers, :topics
```

Comment:
This change seems unnecessary.

Reply:
I know cosmetic pull requests aren't accepted; but while this file was *already* being updated, an ordered list felt more easily searched for the existence of a wanted fixture than a completely random one, and therefore less likely for duplicates/errors in future additions/removals

I can remove it if it's really not acceptable to ever fix a cosmetic issue

Reply:
removed

Reply:
Cleaning up adjacent code as part of a larger change can be acceptable, but in this case I wasn't into it because we don't alphabetise fixture names elsewhere, and the other change to this file would be going away by switching to use the Vegetable class.

Reply:
Makes sense, thanks for the clarification